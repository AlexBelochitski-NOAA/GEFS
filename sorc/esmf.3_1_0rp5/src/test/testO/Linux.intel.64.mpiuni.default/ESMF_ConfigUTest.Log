20190610 112236.240436 INFO     PET0 Running with ESMF Version 3.1.0rp5
20190610 112236.240708 INFO     PET0  Beginning Test, file ESMF_ConfigUTest.F90, line        1415
20190610 112236.240726 INFO     PET0  NUMBER_OF_PROCESSORS           1
20190610 112236.241917 INFO     PET0  PASS  Config Create Test, ESMF_ConfigUTest.F90, line        1423
20190610 112236.242070 INFO     PET0  PASS  Config Destroy Test, ESMF_ConfigUTest.F90, line        1431
20190610 112236.242078 INFO     PET0  Number of failed tests:           0
20190610 112236.242095 INFO     PET0  Ending Test, file ESMF_ConfigUTest.F90, line        1532
