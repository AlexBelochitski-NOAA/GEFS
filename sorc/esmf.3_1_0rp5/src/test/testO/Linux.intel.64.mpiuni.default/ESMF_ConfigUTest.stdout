  Beginning Test, file ESMF_ConfigUTest.F90, line        1415
  NUMBER_OF_PROCESSORS           1
  PASS  Config Create Test, ESMF_ConfigUTest.F90, line        1423
  PASS  Config Destroy Test, ESMF_ConfigUTest.F90, line        1431
  Number of failed tests:           0
  Ending Test, file ESMF_ConfigUTest.F90, line        1532
